deps/internal/client/http_modules.go

```diff
@@ -39,11 +39,12 @@ func (m *httpModuleClient) List(ctx context.Context, in *tracker.ListRequest, op
 }
 
 func (m *httpModuleClient) ListSources(ctx context.Context, in *schema.Module, opts ...grpc.CallOption) (*tracker.ListSourcesResponse, error) {
```

Comment:
The HTTP clients are more or less a dead code path. I wouldn't bother trying to update them.